### PR TITLE
Split clone using consistent snapshot

### PR DESCRIPTION
**Pull request by systay**

### CODE REVIEW

File: go/cmd/vtworkerclient/vtworkerclient.go [hunk 1]
```diff
@@ -55,7 +55,7 @@ func main() {
 			logutil.LogEvent(logger, e)
 		})
 	if err != nil {
-		log.Error(err)
+		log.Errorf("%+v", err)
```
Comment (sougou) [inline go/cmd/vtworkerclient/vtworkerclient.go]: Is this intentional? Or did you change this for troubleshooting?

File: go/vt/worker/restartable_result_reader.go [hunk 2]
```diff
@@ -23,6 +23,8 @@ import (
 	"strings"
 	"time"
 
+	"github.com/golang/glog"
```
Comment (sougou) [inline go/vt/worker/restartable_result_reader.go]: This should instead be `vitess.io/vitess/go/vt/log`. It allows us to remap the logging functions as needed.

File: go/vt/worker/split_clone.go [hunk 3]
```diff
@@ -424,6 +435,29 @@ func (scw *SplitCloneWorker) Run(ctx context.Context) error {
 	return nil
 }
 
+func (scw *SplitCloneWorker) disableUniquenessCheckOnDestinationTablets(ctx context.Context) error {
```
Comment (sougou) [inline go/vt/worker/split_clone.go]: I believe this is still questionable. I don't see it used in this PR. Should we remove it until it becomes official?

File: go/vt/worker/split_clone_cmd.go [hunk 4]
```diff
@@ -82,14 +83,22 @@ const splitCloneHTML2 = `
...
+        <INPUT type="text" id="minHealthyTablets" name="minHealthyTablets" value="{{.DefaultMinHealthyTablets}}"></BR>
       <LABEL for="maxTPS">Maximum Write Transactions/second (If non-zero, writes on the destination will be throttled. Unlimited by default.): </LABEL>
         <INPUT type="text" id="maxTPS" name="maxTPS" value="{{.DefaultMaxTPS}}"></BR>
       <LABEL for="maxReplicationLag">Maximum Replication Lag Seconds (enables the adapative throttler. Disabled by default.): </LABEL>
         <INPUT type="text" id="maxReplicationLag" name="maxReplicationLag" value="{{.DefaultMaxReplicationLag}}"></BR>
       <INPUT type="hidden" name="keyspace" value="{{.Keyspace}}"/>
       <INPUT type="hidden" name="shard" value="{{.Shard}}"/>
+      <LABEL for="useConsistentSnapshot">Use consistent snapshot during the offline cloning:</LABEL>
+        <INPUT type="checkbox" id="useConsistentSnapshot" name="useConsistentSnapshot" value="false"><a href="https://dev.mysql.com/doc/refman/5.7/en/glossary.html#glos_consistent_read" target="_blank">?</a></BR>
```
Comment (mpawliszyn) [inline go/vt/worker/split_clone_cmd.go]: that value needs to be `value="true"`

**Pull request merged**